integreat_cms/cms/templates/messages.html

```diff
@@ -24,3 +26,6 @@
     {% endfor %}
 </div>
 {% endif %}
+<div id="archived-subpages-only" class="bg-blue-100 border-l-4 border-blue-500 text-blue-500 px-4 py-3 mb-5 hidden" role="alert">
+    <p>{% trans 'All subpages have been archived' %}</p>
+</div>
```

Comment:
Since the message template is embedded directly before the normal content, I think we should place this div at the beginning of the content of the page tree view template instead of the end of the messages template...
https://github.com/digitalfabrik/integreat-cms/blob/e76ad67d2e165702acafb089aec676d9203a9b56/integreat_cms/cms/templates/_base.html#L265-L266
Also, I'd probably change the message to "All subpages of this page are archived".